onnx/version_converter/convert.cc

```diff
@@ -0,0 +1,23 @@
+#include "onnx/version_converter/convert.h"
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+namespace {
+  static DefaultVersionConverter _version_converter;
+}
+
+ONNX_NAMESPACE::ModelProto ConvertVersion(
+    const ONNX_NAMESPACE::ModelProto& mp_in,
+    const int target_version) {
+  // Get initial_opsetid from mp_in
+  OpSetID initial_struct(0);
+  for (auto it = mp_in.opset_import().begin(); it != mp_in.opset_import().end(); ++it) {
+    if (it->domain() == "" || it->domain() == "ai.onnx") {
+      initial_struct = OpSetID(*it);
```

Comment:
No need to create a new OpSetID. Just set the opset field should be enough, and then break the loop, since you already find the default domain one